app/controllers/wiki_controller.rb

```diff
@@ -265,6 +265,9 @@ def diff
         helpers.format_text(@diff.content_from.data.text, disable_macro_expansion: true),
         helpers.format_text(@diff.content_to.data.text, disable_macro_expansion: true)
       ).build
+       .gsub(/<ins class="diff(ins|mod)">\n\r?<\/ins>/, '')
+       .gsub(/<del class="diff(del|mod)">\n\r?<\/del>/, '')
```

Comment:
Could you extract this in a dedicated method describing what's being done?

Something like 
```ruby
def remove_empty_newline_highlights(diff)
  diff.gsub(/<ins class="diff(ins|mod)">\n\r?<\/ins>/, "")
      .gsub(/<del class="diff(del|mod)">\n\r?<\/del>/, "")
end